services/ui-src/src/routes/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -1,29 +1,215 @@
         </Heading>
-        <Text>{intro.body}</Text>
+        <Box sx={sx.infoTextBox}>
+          <Text sx={sx.infoHeading}>{intro.infoHeader}</Text>
+          <Text>{intro.info}</Text>
+        </Box>
       </Box>
-    </ReportPage>
+      <Flex sx={sx.submitContainer}>
+        <Button
+          type="submit"
+          colorScheme="colorSchemes.primary"
+          rightIcon={<Icon icon="arrowRight" />}
+          onClick={() => onOpen()}
+          data-testid="submit-button"
+        >
+          {pageLink.text}
+        </Button>
+      </Flex>
+      <Modal
+        actionFunction={() => submitForm()}
+        modalState={{
+          isOpen,
+          onClose,
+        }}
+        content={modal}
+      />
+    </Flex>
   );
 };
 
+interface ReadyToSubmitProps {
+  submitForm: Function;
+  isOpen: boolean;
+  onOpen: Function;
+  onClose: Function;
+}
+
+export const SuccessMessage = ({
+  programName,
+  date,
+  givenName,
+  familyName,
+}: SuccessMessageProps) => {
+  const { intro } = successVerbiage;
+  const utcDateToReadableDate = new Intl.DateTimeFormat("en-US", {
+    dateStyle: "full",
+  }).format(date);
+  const submittedDate = `was submitted on ${utcDateToReadableDate}`;
+  const submittersName = ` by ${givenName} ${familyName}`;
+  return (
+    <Flex sx={sx.contentContainer} data-testid="submitted-view">
+      <Box sx={sx.leadTextBox}>
+        <Heading as="h1" sx={sx.headerText}>
+          <span>
+            <Image src={checkIcon} alt="Checkmark Icon" sx={sx.headerImage} />
+          </span>
+          {intro.header}
+        </Heading>
+        <Box sx={sx.infoTextBox}>
+          <Text sx={sx.infoHeading}>{intro.infoHeader}</Text>
+          <Text>{`MCPAR report for ${programName} ${submittedDate} ${submittersName}`}</Text>
+        </Box>
+      </Box>
+      <Box>
+        <Text sx={sx.additionalInfoHeader}>{intro.additionalInfoHeader}</Text>
+        <Text sx={sx.additionalInfo}>{intro.additionalInfo}</Text>
+      </Box>
+    </Flex>
+  );
+};
+
+interface SuccessMessageProps {
+  programName: string;
+  date: any;
+  givenName?: string;
+  familyName?: string;
+}
+
 const sx = {
+  pageContainer: {
+    width: "100%",
+  },
+  contentContainer: {
+    flexDirection: "column",
+    width: "100%",
+    maxWidth: "reportPageWidth",
+    marginY: "3.5rem",
+    marginLeft: "3.5rem",
+  },
   leadTextBox: {
-    marginBottom: "2.25rem",
+    width: "100%",
+    paddingBottom: "1.5rem",
+    marginBottom: "1.5rem",
+    borderBottom: "1px solid",
+    borderColor: "palette.gray_lighter",
   },
   headerText: {
     marginBottom: "1rem",
-    fontSize: "2rem",
+    fontSize: "4xl",
     fontWeight: "normal",
   },
+  infoTextBox: {
+    marginTop: "2rem",
+  },
+  infoHeading: {
+    fontWeight: "bold",
+    marginBottom: ".5rem",
+  },
+  submitContainer: {
+    justifyContent: "flex-end",
+  },
+  headerImage: {
+    display: "inline-block",
+    marginRight: "1rem",
+    height: "1.72rem",
```

Comment:
Nit: I usually avoid using `rem` in increments smaller than quarters because they're harder to reason about (i.e. off the top of your head, how big is `1.72rem`?). In this case, the difference is less than half a pixel, so we could just use `1.75rem`. Usually you can round to the nearest quarter of a rem but if it has to be precise (for instance maybe it _has to be 27 pixels_ or something) sometimes_ I'll fall back to just directly using pixels because `27px` is obviously not a rem-divisible number and everyone will recognize it has to be a precision thing.